src/main/java/com/github/polimi_mt_acg/back2school/api/v1/auth/AuthenticationResource.java

```diff
@@ -46,7 +46,7 @@ public Response userLogin(LoginRequest request) {
 
     // if the user is null: invalid credentials or user does not exist
     if (user == null)
-      return Response.status(Response.Status.FORBIDDEN).build();
+      return Response.status(Response.Status.BAD_REQUEST).build();
```

Comment:
I see you changed the Response status code. Have you run all the other tests? Are they still running fine? I can't remember whether we had tests checking for status code in case of null user.